src/Raven.Studio/wwwroot/index.html

```diff
@@ -1,60 +1,61 @@
 ﻿<!DOCTYPE html>
 <html>
-    <head>
-        <title>Raven.Studio</title>
+<head>
```

Comment:
can we roll back formatting here? It hurts git blame.

Reply:
Hmmm. I didn't do any formatting myself; Visual Studio must have done this.